app/assets/stylesheets/components-vue/_map.scss

```diff
@@ -125,7 +125,7 @@
 
           position: absolute;
           top: rem-calc(150);
-          right: rem-calc(20);
+          right: 2%;
```

Comment:
So it does work with gutters in the section, but not in the map panel. Interesting...but it will mean that I have to add it to every section element in the views :(